src/Card.css

```diff
@@ -0,0 +1,58 @@
+.card {
+  overflow: hidden;
+  border: solid #00c7d6 5px;
+  border-radius: 10px;
+  box-shadow: 2px 2px 8px 1px grey;
+  width: 100%;
+  height: 250px;
+  color: black;
+  -moz-border-radius: 10px;
+  background-color: #00c7d6;
+  }
+
+.card:hover > .dog-img {
+  opacity: 0.6;
+  filter: blur(1px);
+}
+
+.card:hover > .card-desc {
+  text-decoration: blur;
+}
+
+.card:hover {
+  -webkit-animation-name: wiggle;
+  -webkit-animation-duration: 800ms;
+  -webkit-animation-iteration-count: 1;
+  -webkit-animation-timing-function: ease-in-out;
+}
+
+.dog-img {
+  width: 100%;
+  height: 100px;
+  object-fit: cover;
+  border-radius: 10px;
+}
+
+@-webkit-keyframes wiggle {
+  0% {-webkit-transform: rotate(2deg);}
+  25% {-webkit-transform: rotate(-2deg);}
+  50% {-webkit-transform: rotate(5deg);}
+  75% {-webkit-transform: rotate(-2deg);}
+  100% {-webkit-transform: rotate(0deg);}
+}
+
+/*@-ms-keyframes wiggle {
```

Comment:
Are these keyframes that are being used? If not, can the commented out chunks get removed?

Reply:
you are definitely right, ill be a bit more thorough on my review later in the night or maybe just push them in the morning when im more awake, but i got those taken out for you :)